params/config.go

```diff
@@ -403,6 +405,7 @@ func NewNodeConfig(dataDir string, networkID uint64) (*NodeConfig, error) {
 		log:                   log.New("package", "status-go/params.NodeConfig"),
 		LogFile:               "",
 		LogLevel:              "ERROR",
+		NoDiscovery:           true,
```

Comment:
In this case, I think we can remove https://github.com/status-im/status-go/blob/develop/cmd/node-canary/main.go#L257

Reply:
It's done.